cypress-io/cypress
# chore: move snap-shot-it@7.9.10 patch version

- closes https://github.com/cypress-io/cypress/issues/29647

### Additional details

Building Cypress from source with Yarn resulted in a warning from patch-package

> Warning: patch-package detected a patch file version mismatch
>  Patch file created for
>
>    snap-shot-it@7.9.3
>
>  applied to
>
>    snap-shot-it@7.9.10

### Steps to test

Execute:

```shell
git clean -xfd
yarn
```

and confirm that no warnings from `patch-package` are produced

### How has the user experience changed?

This issue affects contributors only.
This PR removes a warning from `patch-package` when building Cypress from source.

### PR Tasks

- [na] Have tests been added/updated?
- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)?
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
